install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -108,7 +108,7 @@ plug-ins that are supported. The following sections describe the basic object
 definition for a _StorageClass_ and specific examples for each of the supported
 plug-in types.
 
-[[basic-spec-defintion]]
+[[basic-spec-definition]]
```

Comment:
@tmorriso-rh - make sure to check that the old anchor tag is not being used in the docs. If it is, then those docs would need changing as well.

Reply:
Yes, I checked these during the build and they are working.